suricata/update/main.py

```diff
@@ -381,6 +381,11 @@ def fetch(self, url):
                     "will use latest cached version: %s", url, err)
                 return self.extract_files(tmp_filename)
             raise err
+        except IOError as err:
+            sys.stdout.write("\n")
```

Comment:
Oops, a  little mistake here!
I think I should call `progress_hook_finish(self)` here instead of explicitly adding `sys.stdout.write("\n")`. 
Would that be fine @shivan1b?

Reply:
Works either way. I checked on my side, confirm it on your too that it works under the conditions that have been taken care of in `progress_hook_finish`.

Reply:
It works.